### DISCUSSION TITLE
Elaboration on steps to becoming a maintainer

### DISCUSSION THREAD

**Discussion by CBID2** [open-sauced/feedback#14] [General feedback or Bugs]
## Problem
I was looking through the [Maintainer Guide](https://docs.opensauced.pizza/maintainers/setting-up-a-new-repository/). While saying "Show Your Pizza worth" is humorous and encouraging, people that have been contributing to OpenSauced on a regular basis and want to transition into that role might wonder when they will be picked. 

## Possible Solution
I think there could be a set amount of months a person contributes albeit review or pull requests. Something similar to [MDN Web Notes' standards for maintainers](https://developer.mozilla.org/en-US/docs/MDN/Community/Roles_teams).

**Comment by bdougie** [comment]
I think there is a conversation to be had here. Steps on how to join the Triage team could be more precise and the path from Triage to Maintainer could be organized into thoughts. I don't have that answer, but happy to hear ideas. 

Perhaps we can look to the k8s SIGs (Kubernetes Special Interest Groups) - https://github.com/kubernetes-sigs
